src/main/resources/page/js/css-and-js-files.js

```diff
@@ -5,17 +5,23 @@ function externalFilesLoader(){
     addCssFiles();
 }
 
-function addJsFiles(){
+function addJsFiles() {
+    jsLoader('home-page-settings.js');
+    jsLoader('header.js');
+    jsLoader('footer.js');
+    jsLoader('home-page-loader.js');
```

Comment:
Will every JS files necessary to be loaded in every pages? Or there will be separate functions for it in the future?

Reply:
It is necessary, but it will be separate functions in the future.